addons/mail/static/tests/discuss/message_pin/pinned_messages.test.js

```diff
@@ -13,26 +13,6 @@ import { disableAnimations } from "@odoo/hoot-mock";
 describe.current.tags("desktop");
 defineMailModels();
 
-test("Pin message", async () => {
```

Comment:
I don't think we want to move all our tests in other modules "just in case". It would be nice if we could find a solution to run the test suite with just mail, or with all modules installed.